### PR TITLE
footer changes squashing

### PR DESCRIPTION
**Pull request by AdelineA**

### CODE REVIEW

File: ReadMe.md
Comment (gsngdavid) [inline ReadMe.md:1]: Remember to shift terminal history to main branch

File: home.html
Comment (gsngdavid) [inline home.html:1]: 👌

**Review by gsngdavid — APPROVED** [review]
👍

File: ReadMe.md
Comment (AdelineA) [reply ReadMe.md:1]: Thank you, I'll work on that.

**Review by Ntwali-Josue — APPROVED** [review]
Can you fix your conflicts in the `README.md`, and then merge the PR?

**Pull request merged**